modules/minimum-required-permissions-gcp-upi.adoc

```diff
@@ -0,0 +1,307 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-account.adoc
+// * installing/installing_gcp/installing-gcp-user-infra.adoc
+// * installing/installing_gcp/installing-restricted-networks-gcp.adoc
+
+[id="installation-gcp-permissions-upi_{context}"]
+= Required GCP permissions for user-provisioned infrastructure
+
+To deploy and destroy an {product-title} cluster with user-provisioned infrastructure, the service account requires the following permissions.
+
+.Required permissions for creating network resources
+[%collapsible]
+====
+* `compute.addresses.create`
+* `compute.addresses.createInternal`
+* `compute.addresses.get`
+* `compute.addresses.list`
+* `compute.addresses.use`
+* `compute.addresses.useInternal`
+* `compute.firewalls.create`
+* `compute.firewalls.get`
+* `compute.firewalls.list`
+* `compute.forwardingRules.create`
+* `compute.forwardingRules.get`
+* `compute.forwardingRules.list`
+* `compute.forwardingRules.setLabels`
+* `compute.networks.create`
+* `compute.networks.get`
+* `compute.networks.list`
+* `compute.routers.create`
+* `compute.routers.get`
+* `compute.routers.list`
+* `compute.routers.update`
+* `compute.routes.list`
+* `compute.subnetworks.create`
+* `compute.subnetworks.get`
+* `compute.subnetworks.list`
+* `compute.subnetworks.use`
+* `compute.subnetworks.useExternalIp`
+====
+
+.Required permissions for creating network resources during installation
+[%collapsible]
+====
+* `dns.changes.create`
+* `dns.changes.get`
+* `dns.managedZones.create`
+* `dns.managedZones.delete`
+* `dns.managedZones.get`
+* `dns.managedZones.list`
+* `dns.networks.bindPrivateDNSZone`
+* `dns.resourceRecordSets.create`
+* `dns.resourceRecordSets.delete`
+* `dns.resourceRecordSets.list`
+* `dns.resourceRecordSets.update`
```

Comment:
With my suggestions on the IPI part, suggest to move this permission to the section "Required permissions for creating DNS resources", and remove the section "Required permissions for creating network resources during installation".

Reply:
Addressed